azure/table_azure_storage_account_local_user.go

```diff
@@ -0,0 +1,248 @@
+				Name:        "is_nfsv3_enabled",
+				Description: "Indicates if the local user is enabled for access with NFSv3 protocol.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("Properties.IsNFSv3Enabled"),
+			},
+			{
+				Name:        "sid",
+				Description: "A unique Security Identifier that is generated by the server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Sid"),
+			},
+			{
+				Name:        "user_id",
+				Description: "A unique Identifier that is generated by the server.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("Properties.UserID"),
+			},
+			{
+				Name:        "allow_acl_authorization",
+				Description: "Indicates whether ACL authorization is allowed for this user.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("Properties.AllowACLAuthorization"),
+			},
+			{
+				Name:        "extended_groups",
+				Description: "Supplementary group membership.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Properties.ExtendedGroups"),
+			},
+			{
+				Name:        "group_id",
+				Description: "An identifier for associating a group of users.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("Properties.GroupID"),
+			},
+			{
+				Name:        "permission_scopes",
+				Description: "The permission scopes of the local user.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Properties.PermissionScopes"),
+			},
+			{
+				Name:        "ssh_authorized_keys",
+				Description: "Local user ssh authorized keys for SFTP.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Properties.SSHAuthorizedKeys"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ID").Transform(idToAkas),
+			},
+
+			// Azure standard columns
+			{
+				Name:        "resource_group",
+				Description: ColumnDescriptionResourceGroup,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID").Transform(extractResourceGroupFromID),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listStorageAccountLocalUsers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// Get the storage account details from the parent hydrate
+	storageAccount := h.Item.(*armstorage.Account)
+	resourceGroup := ""
+	if storageAccount.ID != nil {
+		parts := strings.Split(*storageAccount.ID, "/")
+		for i := 0; i < len(parts)-1; i++ {
+			if strings.EqualFold(parts[i], "resourceGroups") {
+				resourceGroup = parts[i+1]
+				break
+			}
+		}
+	}
```

Comment:
Can we optimize the code to extract the `resourceGroups` value by simply splitting the ID and retrieving the resource group name from a specific index, similar to how it's handled in other tables?